+++
# Date this page was created.
date = "2018-09-03"

# Project title.
title = "Open Positions"

# Project summary to display on homepage.
summary = "Join us"

# Optional image to display on homepage (relative to `static/img/` folder).
image_preview = "people/plus_square.jpg"
image = "people/plus_square.jpg"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["Group Leader"]

# Optional external URL for project (replaces project detail page).
#external_link = "#about"

# Does the project detail page use math formatting?
math = false

# List your qualifications (such as academic degrees).
# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

[[social]]
icon = "envelope"
icon_pack = "fa"
link = "mailto:gioele.lamanno@epfl.ch"
 
+++
 
# Openings and available projects

## Postdoctoral and PhD student positions

<div style="text-align: justify">

There are no open positions at the moment.

Outstanding candidates might still be considered, to enquires please contact me at <a href="mailto:gioele.lamanno@epfl.ch">gioele.lamanno@epfl.ch</a>.<br><br>

For Ph.D. students, while direct enquiries are appreciated, to become a Ph.D. student in the lab an official application needs to go through official channels of EPFL-SV doctoral schools: <a href="https://www.epfl.ch/education/phd/edne-neuroscience/edne-how-to-apply/">EDNE</a>, <a href="https://www.epfl.ch/education/phd/edcb-computational-and-quantitative-biology/edcb-how-to-apply/">EDCB</a> or <a href="https://www.epfl.ch/education/phd/edms-molecular-life-sciences/edms-how-to-apply/">EDMS</a>. PhD recruitment happens only twice per year, so if interested please keep an eye on the application deadlines.<br><br>

For postdoctoral researcher positions, please send your request as a single PDF file – including a CV, a complete list of publications, a statement of research interests, and the contact information of at least two reference persons – to <a href="mailto:gioele.lamanno@epfl.ch">Gioele</a>.
</div>

## Masters Projects

We always offer a set of exciting computational and experimental projects for Masters student interested to do a thesis or a Semester project in our lab. Example of projects available are listerd below. Contact <a href="mailto:gioele.lamanno@epfl.ch">Gioele</a>

<div style="text-align: justify">
**Computational Projects**

Particularly indicated for EPFL students of Data Science and Computer science.

Title> **Integration, modeling and visualization of central nervous systems data**<br>
**Project Description >**<br>
**- Background**: The central nervous system is complex and its generation involves several mechanisms and parameters that control it. To date, we lack an integrative modeling of these data to visualize and better understand the process that control neuronal type-specific generation, differentiation, and circuit assembly. For this project we will use the cerebellum, a critical regulator of motor function.<br>
**- Activities**:  The student will: (1) Integrate datasets within a common structure from different data sources (2) Make this data easily explorable through data visualization tools (for example using R Shiny apps) (3) Build integrative models this datasets through time to explore cerebellar development.<br>
**Faculty and laboratory >** - SV - BMI - Neurodevelopmental Systems Biology Lab, in collaboration with the group of Ludovic Telley at Unil<br>
**Available to>** Master students of Data Science, Computer science and Life Science Engineering. Knowledge of both python and R is required.<br>

Title> **Extensions of the RNA velocity algorithm**<br>
**Project Description >**<br>
**- Background**: An ambitious goal of single-cell analyses is describing dynamical biological processes and shedding light on gene regulation mechanisms. The challenge in studying these phenomena consists on the destructive nature of the measurement that can only provide a static snapshot of the single cell states. to overcome this fundamental limit of single cell technologies, I recently developed a novel method, named “RNA velocity” as it estimates the first derivative of gene expression for each gene in a cell (RNA velocity of single cells, Nature 2018). The core idea is that measuring the abundance of both unspliced and spliced RNA in the same cell, we can estimate the rate of change of gene expression and predict the future expression levels of a single cell.<br>
**- Activities**: This projects will start by introducing a series of improvement to the current RNA velocity algorithm(1)  to better adapt it to different kind of datasets and (2) to provide statistical confidence interval around the estimation. The student will also (3) explore possibilities of a complete reformulation of the algorithm as a convex optimisation problem and/or statistical bayesian hierarchical model, using state of the art libraries (PyTorch, cvxpy) and modelling languages (Stan).<br>
**- Reference**: [RNA velocity of single cells. La Manno et al. Nature 2018](https://www.nature.com/articles/s41586-018-0414-6)<br>
**Available to>** Master students of Data Science, Computer science and Life Science Engineering
**Faculty and laboratory >** - SV - BMI - Neurodevelopmental Systems Biology Lab<br>
**Contact person >** Gioele La Manno - <gioele.lamanno@epfl.ch><br> <br>

**Experimental Projects**

Title> **Identification of different population of neural progenitors by spatial transcriptomics**<br>
**- Background**: During the development of the nervous system, an initial stem-cell pool specifies in hundreds of different neuronal cell types through a highly regulated gene regulation programs. Single-cell RNA sequencing profiling has allowed the transcriptomic characterization of many of these lineages, and importantly the definition of the sequence of intermediate states that eventually leads to the final adult cell types. To understand how gene regulation programs unfold and how mature cell forms, it is essential to characterize the many intermediate states further. Beyond measuring their transcriptome it is important to localize those cells in the tissue to start understanding their function<br>
**- Project**: This project aims at the identification of spatial localization of different populations of radial-glial progenitors. The student will: (1) Design a set of in-situ hybridization probes, starting from single-cell data. (2) Validate the FISH probes. (3) Collect images across the ventricular zone of mouse embryos of different ages. (4) Perform data analysis to integrate the information collected.
